### DISCUSSION TITLE
《关于my homi 一直在反复表达自己是有地将臣这件事》

### DISCUSSION THREAD

**Discussion by biejian-yuanshen** [share121/inter-knot#1179] [其他]
![qq_pic_merged_1723311681759](https://github.com/user-attachments/assets/5c5a7be3-0ae0-4138-a3dd-a2cdf9dc2456)

是这样的我的朋友在互联网和我的影响下日渐沉迷上了日式galagame美少女游戏，且完全无法自拔，颇有朕当年的姿态，其癫狂程度已经来到了一日之内说要说自己是各种男主角十几次的地步，我很担心他，我现在有一个计划就是把他变成homo这样他的病情估计就会好转，各位是什么看法呢，可以在楼下留留言

**Comment by KawaiiSparkle** [comment]
可能得看体型、年龄啥的（？）我感觉我不是很懂（

**Reply by biejian-yuanshen** [reply]
体型瘦小犹如佝偻

**Reply by KawaiiSparkle** [reply]
怂恿他变成小南梁

**Comment by yuki15532-cpu** [comment]
这是病情加重了吧。等等...难道你是homo，想做他先辈？！

**Comment by 290678** [comment]
我不知道，但我要Ciallo～(∠・ω< )⌒☆
